assets/js/blocks/order-confirmation/billing-wrapper/edit.tsx

```diff
@@ -0,0 +1,39 @@
+/**
```

Comment:
If we run with this solution and have a billing version and a shipping version, do you think it would be possible to handle the conditional display logic in the PHP render method? That might be a clean approach.